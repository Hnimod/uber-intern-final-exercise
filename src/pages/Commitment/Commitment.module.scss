.overlay {
  background-color: rgba(255, 255, 255, 0.9);
  max-width: 90rem;
  height: 100%;
  padding: 2rem;
  margin: 0 auto;
  flex: 1;
}

.content {
  margin-top: 1rem;
}

.description {
  line-height: 25px;
  margin-bottom: 3rem;
}

.guide {
  & > h3:not(:first-child) {
    margin-top: 2rem;
  }

  & > p {
    margin-top: 1rem;
    word-spacing: 3px;
    line-height: 30px;
    color: rgb(88, 88, 88);
  }
}
