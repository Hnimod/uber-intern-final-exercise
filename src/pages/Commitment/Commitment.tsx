import BackgroundImage from '../../layouts/BackgroundImage';

import styles from './Commitment.module.scss';

const Commitment = () => {
  return (
    <BackgroundImage>
      <div className={styles.overlay}>
        <h1 className={styles.title}>
          At Uber we care about you <br></br>& the trust you give use
        </h1>
        <div className={styles.content}>
          <p className={styles.description}>
            Uber puts transportation at your fingertips. Understanding how and
            why we use your information should be just as easy. Our Privacy
            Policy and Privacy FAQs provide the latest information about the
            data we collect and how we use it. We also want you to know a few
            more things about our approach to privacy:
          </p>

          <div className={styles.guide}>
            <h3>Your data works for you</h3>
            <p>
              We use your data to provide convenient transportation and delivery
              options, to help drivers amximize their earnings, and to protect
              the safety and security of our users. And we dont't rent or sell
              your data - to anyone
            </p>
            <h3>Safety first</h3>
            <p>
              Your data enables us to determine the safest pick-ups and
              drop-offs, deter unsafe driving habits, help you avoid
              accident-prone roads, and to develop new features to get where
              you're going, safe and sound
            </p>
            <h3>You are in control</h3>
            <p>
              Through the settings in the Uber app or on you device, you can
              choose when to share your location, sync your contacts and
              calendar, personalize your app, and even delete you account
            </p>
          </div>
        </div>
      </div>
    </BackgroundImage>
  );
};

export default Commitment;
