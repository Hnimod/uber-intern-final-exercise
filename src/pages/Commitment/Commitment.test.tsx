import React from 'react';
import Commitment from './Commitment';
