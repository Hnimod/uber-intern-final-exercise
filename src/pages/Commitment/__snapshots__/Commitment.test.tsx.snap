// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render commitment page correctly 1`] = `
<BackgroundImage>
  <div
    className="overlay"
  >
    <h1
      className="title"
    >
      At Uber we care about you 
      <br />
      & the trust you give use
    </h1>
    <section
      className="content"
    >
      <p
        className="description"
      >
        Uber puts transportation at your fingertips. Understanding how and why we use your information should be just as easy. Our Privacy Policy and Privacy FAQs provide the latest information about the data we collect and how we use it. We also want you to know a few more things about our approach to privacy:
      </p>
      <div
        className="guide"
      >
        <h3>
          Your data works for you
        </h3>
        <p>
          We use your data to provide convenient transportation and delivery options, to help drivers amximize their earnings, and to protect the safety and security of our users. And we dont't rent or sell your data - to anyone
        </p>
        <h3>
          Safety first
        </h3>
        <p>
          Your data enables us to determine the safest pick-ups and drop-offs, deter unsafe driving habits, help you avoid accident-prone roads, and to develop new features to get where you're going, safe and sound
        </p>
        <h3>
          You are in control
        </h3>
        <p>
          Through the settings in the Uber app or on you device, you can choose when to share your location, sync your contacts and calendar, personalize your app, and even delete you account
        </p>
      </div>
    </section>
  </div>
</BackgroundImage>
`;
